src/edit/RotateHandle.js

```diff
@@ -16,9 +16,13 @@ L.RotateHandle = L.EditHandle.extend({
 
 		overlay.editing._hideToolbar();
 
-		overlay.editing._rotateBy(angle);
+		// overlay.editing._rotateBy(angle);
```

Comment:
Just a heads up, this comment and the code below are identical. I guess such issues will be fixed in the cleanup later like you suggested?
```suggestion
```